examples/custom-theme/package.json

```diff
@@ -8,8 +8,7 @@
     "kepler.gl": "^1.0.0-2",
     "react": "^16.8.4",
     "react-dom": "^16.8.4",
-    "react-palm": "^1.1.2",
-    "react-redux": "^4.0.0",
```

Comment:
we don't need this dependency since we are using __enhanceReduxMiddleware__ which will inject react-palm into application consumer